README.md

```diff
@@ -6,35 +6,51 @@ Try the current version at: [https://cs121-team-panda.github.io/coconut-interpre
 
 Developed by Jonathan Cruz, Teerapat Jenrungrot, Natalie Kadonaga, and Brittany Wang.
 
+## Installation
+```pip install -r requirements.txt```
+
+## Running the app
+Requires Python 3 [(recent Python version here)](https://www.python.org/downloads/). 
```

Comment:
nit: Up to you but we don't really need to link to Python since people get it from other sources such as brew and Anaconda